src/Console/Command/FixCommandHelp.php

```diff
@@ -205,6 +205,13 @@ public static function getHelpCopy()
 
     $ vendor/bin/php-cs-fixer fix --config=.php_cs.dist -v --dry-run --using-cache=no --path-mode=intersection `git diff --name-only --diff-filter=ACMRTUXB \$COMMIT_RANGE`
 
+If filenames contain spaces:
```

Comment:
as this condition is not fixed per repo (as one could add file with whitespace after integrating PHP CS Fixer), I would drop previous solution, as it's buggy